salt/modules/dockermod.py

```diff
@@ -6786,3 +6786,58 @@ def sls_build(repository,
         stop(id_)
         rm_(id_)
     return ret
+
+
+def prune(system=True, containers=False, networks=False, images=False, build=False, volumes=False, **filters):
+    '''
+    Prune Docker's various subsystems
+
```

Comment:
Doing a quick sweep of the codebase, these API functions all seem to have been added in docker-py 2.1.0. Can you please add the following here?

```rst
    .. note::
        This requires docker-py version 2.1.0 or later.
```

If pre-2.1.0 docker-py is installed, `_client_wrapper()` function will handle the API error from the unknown function call, so we won't have a traceback. This is just a bit of extra information that should help users who may be on earlier docker-py releases.